# For some reason, `--disable-sse2` did not work for aarch64. So completely disable it from configure.
diff --git a/configure.in b/configure.in
index 4848e11..947d5c2 100644
--- a/configure.in
+++ b/configure.in
@@ -74,18 +74,6 @@ AC_ARG_ENABLE(
     [CFLAGS="-DPROFILE -pg ${CFLAGS}"]
 )
 
-dnl ------------------------------------------------------------------
-dnl Checks for SSE2 build
-dnl ------------------------------------------------------------------
-AC_ARG_ENABLE(
-    sse2,
-    [AS_HELP_STRING(
-        [--enable-sse2],
-        [enable SSE2 optimization routines]
-        )],
-    [CFLAGS="-msse2 -DUSE_SSE ${CFLAGS}"]
-)
-
 dnl ------------------------------------------------------------------
 dnl Checks for library functions.
 dnl ------------------------------------------------------------------
